test/html/multiple-imports.html

```diff
@@ -7,5 +7,6 @@
     Code distributed by Google as part of the polymer project is also
     subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+<link rel="import" href="multiple-imports.html">
```

Comment:
I'm having trouble understanding the issue / when this bug appears. Does a file have to import itself? Isn't that already an error?